tasks/libs/datadog_api.py

```diff
@@ -0,0 +1,27 @@
+def create_count(metric_name, timestamp, value, tags):
```

Comment:
Should we calculate the timestamp inside this function?

Reply:
I want to ensure that all the metrics I'm creating in a run of the task have the same timestamp, which is why I opted for creating the timestamp beforehand and passing it here.

Reply:
Okay, that makes sense